src/BlocklyInterface/blocklySlice.ts

```diff
@@ -138,7 +138,9 @@ export const saveBlocklyState = (state: RootState) => {
   try {
     const serializedState = JSON.stringify(state.blockly);
     localStorage.setItem(localStorageKey, serializedState);
-  } catch {}
+  } catch {
+    /* empty */
+  }
```

Comment:
Note: I know this is pre-existent code :). 

`stringify` will only throw if it finds a circular dependency. And `setItem` will only throw if the storage quota is exceeded.  In any of the cases, something fishy might be happening with the app if this throws. So let's not consume it. 

```suggestion
  } catch (err) {
    console.error(err)
  }
```

Or, **even better**, let's not catch it and make the app nicely crash.

Reply:
yup that make sense, I was just did some quick lint fixing but didn't put too much thinking around the implementation